iotdb/src/main/java/org/apache/iotdb/db/utils/OpenFileNumUtil.java

```diff
@@ -38,7 +39,7 @@
   private static final int PID_ERROR_CODE = -1;
   private static final int UNSUPPORTED_OS_ERROR_CODE = -2;
   private static final int UNKNOWN_STATISTICS_ERROR_CODE = -3;
-  private static final String IOTDB_PROCESS_KEY_WORD = "iotdb.IoTDB";
```

Comment:
You should not change it because in our startup script, IoTDB process is named as "iotdb.IoTDB" 
![image](https://user-images.githubusercontent.com/8021527/53391829-893e7d00-39d2-11e9-8601-f8109860f026.png)

Reply:
The previous keyword ```iotdb.IoTDB```  is ineffective on MacOS, however keyword ```IoTDB``` is working on both Linux and MacOS, which is why I change the keyword to ```IoTDB```.

Reply:
If so, remember to modify the script file.